src/tokens/solana.tokenlist.json

```diff
@@ -10528,11 +10528,28 @@
       "extensions": {
         "website": "https://www.mercurial.finance/"
       }
+    },
+    {
+      "chainId": 101,
+      "address": "8kRacWW5qZ34anyH8s9gu2gC4FpXtncqBDPpd2a6DnZE",
+      "symbol": "MECA",
+      "name": "Coinmeca",
+      "decimals": 9,
+      "logoURI": "https://firebasestorage.googleapis.com/v0/b/coinmeca-3e733.appspot.com/o/Coinmeca.svg?alt=media&token=4ef710cb-df4c-47aa-9614-c867153db029",
+      "tags": [
+        "utility-token"
+      ],
+      "extensions": {
+        "website": "https://coinmeca.net/",
+        "medium": "https://coinmeca.medium.com/",
+        "twitter": "https://twitter.com/coinmeca",
+        "telegram": "https://t.me/coinmeca",
```

Comment:
Try removing the trailing comma

Reply:
You are star! My mistake - let me fix it quickly!
Many thanks again!

Reply:
It's again waiting for approval - sorry for inconvenience

Reply:
Or @bartosz-lipinski could you please approve the waiting 2 workflows?
Many thank

Reply:
@mvines could you please press again the same button?

Reply:
Many thanks